src/main/java/org/osc/sdk/manager/element/SecurityGroupInterfaceElement.java

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.sdk.manager.element;
+
+import java.util.Set;
+
+public interface SecurityGroupInterfaceElement {
+
+	/**
+	 * Id of the security group interface.
```

Comment:
for simple things like this u can remove the description part and simply just have the return part below. Having both with the same or very similar information is redundant.